package.json

```diff
@@ -14,7 +14,8 @@
     },
     "categories": [],
     "activationEvents": [
-        "*"
+        "workspaceContains:**/*.story",
```

Comment:
Is there a way to activate on an open file?

Reply:
yes https://code.visualstudio.com/api/references/activation-events